src/api/sensorData.proto

```diff
@@ -2,12 +2,24 @@ syntax = "proto2";
 
 package api;
 
+message Location {
+        optional float longitude = 1;
+        optional float latitude = 2;
+        optional int32 altitude = 3;
+}
+
 message SensorData {
         optional bool motion = 1;
         optional uint32 water = 2;
         optional float temperature_celcius = 3;
         optional float temperature_fahrenheit = 4;
         optional float humidity = 5;
 
+        optional Location location = 6;
+        optional float water_distance = 7;
+        optional bool not_raining = 8;
+        optional float pump_activated_timer = 9;
+        optional bool pump_activated = 10;
```

Comment:
Let's make this more generic;

- `level`
- `triggered`
- `actuator_active`
- `actuator_time`

Note that the last two fields doesn't sound as if they have anything to do with `SensorData`. Maybe rename the whole thing to `DeviceData`?